project/AkkaDependency.scala

```diff
@@ -14,7 +14,7 @@ object AkkaDependency {
   // else if akka version is "default", then the hard coded default will be used (jenkins doesn't allow empty values for config axis)
   // else if akka.version is anything else, then the given version will be used
 
-  val defaultAkkaVersion = "2.5.19"
+  val defaultAkkaVersion = "2.5.22"
```

Comment:
We haven't updated the required version lightly in the past. Do we care now?

Reply:
Because 2.5.22 is the first (and currently only) version that has a release for 2.13.0-RC1.

Could also do a cross-scala-variant thing with this (but that could get confusing if you forget)

Reply:
That's the question if we care enough to keep 2.5.19 for 2.11/2.12 but I guess it doesn't matter a lot.